meta-resin-common/recipes-bsp/u-boot/patches/resin-specific-env-configuration.patch

```diff
@@ -52,17 +57,33 @@ index 0000000..ef8a2b0
 +		"echo Import ${resin_env_file} in environment;" \
 +		"env import -t ${resin_kernel_load_addr} ${filesize}\0" \
 +	\
++	"resin_flasher_detect=" \
++		"fatload mmc " __stringify(RESIN_EXTERNAL_MMC) ":${resin_boot_part} ${resin_kernel_load_addr} ${resin_flasher_external_flag_file};\0" \
```

Comment:
This does not work as expected. When used below, in the:

"if run resin_flasher_detect; then " \
 ++				"setenv resin_mmcdev " __stringify(RESIN_EXTERNAL_MMC) "; "\
 ++			"else; " \
 ++				"setenv resin_mmcdev " __stringify(RESIN_INTERNAL_MMC) "; "\

it will always go through the else branch.

You can instead use the -e flag for the test command and then the above line would become:

++	"resin_flasher_detect=" \
 ++		"test -e mmc " __stringify(RESIN_EXTERNAL_MMC) ":${resin_boot_part} ${resin_flasher_external_flag_file};\0" \